packages/standard-prettier/index.js

```diff
@@ -1,3 +1,7 @@
 module.exports = {
   plugins: ["prettier"],
-};
+  printWidth: 120,
+  trailingComma: "all",
+  singleQuote: false,
+  semi: false,
```

Comment:
why false?

Reply:
i think semicolon off is just cleaner, no strong preference

Reply:
I think we can keep the default. We can't remove the semi colon. This should be reinforced. Remember this issue - https://cakedefi.slack.com/archives/C05DS574JS0/p1690452435684489